tests/test_update_config.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+import pytest
+
+
+
+@pytest.mark.parametrize(
+    "conf_object",
+    [{"A": 1},
```

Comment:
I think we can put these parameters to variables or something like that, right now, is very hard to read.

Reply:
But pytests docs tells to parametrize test function like that.  
Or I do not understand what do You meen ???

Reply:
I meant to put these parameters into variables with the correct name.

e.g
```
my_parameter = {"A": 1}
my_second_parameter =  type("C", (), {"A": 1})
my_third_parameter = pytest.param(str(Path(__file__).parent / "static/app_conf.py"), marks=...)

 [my_parameter,
     my_second_parameter,
     my_third_parameter],
```

Reply:
OK.